DataFormats/L1THGCal/interface/HGCFETriggerDigi.h

```diff
@@ -51,7 +51,7 @@ namespace l1t {
     template<typename IDTYPE>
     IDTYPE getDetId() const { return IDTYPE(detid_); }
     template<typename IDTYPE>
-    void   setDetId(const IDTYPE& id) { detid_ = id.uint32_t(); }
+    void   setDetId(const IDTYPE& id) { detid_ = id(); }
```

Comment:
This is wrong (it calls the codec's id() function).
All DetIds have the uint32_t() function: https://github.com/cms-sw/cmssw/blob/CMSSW_6_2_X_SLHC/DataFormats/DetId/interface/DetId.h#L42